src/fidesops/service/connectors/http_connector.py

```diff
@@ -78,13 +78,7 @@ def retrieve_data(
         privacy_request: PrivacyRequest,
         input_data: Dict[str, List[Any]],
     ) -> List[Row]:
-        """Retrieve data in a connector dependent way based on input data.
-
-        The input data is expected to include a key and list of values for
-        each input key that may be queried on.
-
-        TODO: implement when HTTPS Connectors can be part of the traversal.
-        """
+        """Currently not supported as webhooks are not called at the collection level"""
```

Comment:
I know this is here because it is required by the `ABC` so you can't remove it, so maybe raise an exception if called because it is still callable.

```suggestion
# or whatever exception you think is appropriate
class NotSupportedError(Exception):
    pass
    
    def retrieve_data(
        self,
        node: TraversalNode,
        policy: Policy,
        privacy_request: PrivacyRequest,
        input_data: Dict[str, List[Any]],
    ) -> List[Row]:
        """Currently not supported as webhooks are not called at the collection level"""
        
        raise NotSupportedError("Currently not supported as webhooks are not called at the collection level")
```

Reply:
I can do that - webhooks shouldn't be called as part of the graph currently because there's no dataset configs attached to the https connection configs, but this would help cover our bases.